dalite_xblock/public/js/dalite_xblock.js

```diff
@@ -4,8 +4,97 @@ function DaliteXBlock(runtime, element) {
     // hack to make LTI Consumer css applied to Dalite XBlock
     $(element).addClass("xblock-student_view-lti_consumer xblock-student_view");
     $(element).children(".xblock-dalite").addClass("lti_consumer");
-    
+
+    $(function ($) {
```

Comment:
@pomegranited  Frankly I could have just register three events (there are three ways to close the modal). But I decided to copy&rename modal from `lti-consumer` as: I felt this was too much coupling, if anytime in the future HTML layout of the modal would be changed, or some class be renamed, we'd have a chance of silent failure on a feature that is easy to overlook while testing in a project that has no integration tests yet. Now we "own" this code and it won't get broken by upstream changes. 

Maybe there is better argumentation that will convince me to the opposite :)